types/stripe/index.d.ts

```diff
@@ -4903,6 +4848,258 @@ declare namespace Stripe {
+            customer: string | customers.ICustomer | null;
+
+            /** Has the value true if the object exists in live mode or the value false if the object exists in test mode. */
+            livemode: boolean;
+
+            /** Set of key-value pairs that you can attach to an object. This can be useful for storing additional information about the object in a structured format. */
+            metadata: IMetadata;
+        }
+
+        interface ICardPaymentMethod extends IBasePaymentMethod {
+            type: "card";
+            card: {
+                /** Card brand. Can be `amex`, `diners`, `discover`, `jcb`, `mastercard`, `unionpay`, `visa`, or `unknown`. */
+                brand: CardBrand;
+
+                /** Check results by Card networks on Card address and CVC at time of payment. */
+                checks: CardChecks;
+
+                /**
+                 * Two-letter ISO code representing the country of the card. You could use this attribute to get a sense of
+                 * the international breakdown of cards you’ve collected.
+                 */
+                country: string;
+
+                /** Two-digit number representing the card’s expiration month. */
+                exp_month: number;
+
+                /** Four-digit number representing the card’s expiration year. */
+                exp_year: number;
+
+                /**
+                 * Uniquely identifies this particular card number. You can use this attribute to check whether two
+                 * customers who’ve signed up with you are using the same card number, for example.
+                 */
+                fingerprint: string;
+
+                /** Card funding type. Can be credit, debit, prepaid, or unknown. */
+                funding: "credit" | "debit" | "prepaid" | "unknown";
+
+                /** Details of the original PaymentMethod that created this object. */
+                generated_from: null | {
+                    /** The charge that created this object. */
+                    charge: string;
+
+                    /** Transaction-specific details of the payment method used in the payment. */
+                    payment_method_details: charges.IPaymentMethodDetails;
+                };
+
+                /** The last four digits of the card. */
+                last4: string;
+
+                /** Contains details on how this Card may be be used for 3D Secure authentication. */
+                three_d_secure_usage: {
+                    /** Whether 3D Secure is supported on this card. */
+                    supported: boolean;
+                };
+
+                /** If this Card is part of a card wallet, this contains the details of the card wallet. */
+                wallet: null | CardWallet;
+            };
+        }
+
+        interface ICardPresentPaymentMethod extends IBasePaymentMethod {
```

Comment:
The documentation for this Payment Method type is empty: https://stripe.com/docs/api/payment_methods/object#payment_method_object-card_present